src/preview.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * A decorator is a way to wrap a story in extra “rendering” functionality. Many addons define decorators
+ * in order to augment stories:
+ * - with extra rendering
+ * - gather details about how a story is rendered
+ *
+ * When writing stories, decorators are typically used to wrap stories with extra markup or context mocking.
+ *
+ * https://storybook.js.org/docs/react/writing-stories/decorators#gatsby-focus-wrapper
+ */
+import { PARAM_KEY } from "./constants";
+import { withGlobals } from "./withGlobals";
+import { withRoundTrip } from "./withRoundTrip";
+
+export const decorators = [withGlobals, withRoundTrip];
```

Comment:
@winkerVSbecks does this work with the new default export? If so, we should probably standardize on it:

```js
export default { 
  decorators: ...,
  globals: ...,
}
```

If not, we should probably fix it on the storybook side & then update this

Reply:
It works!